modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/tests/FDSCustomView.testcase

```diff
@@ -102,6 +102,55 @@ definition {
 		}
 	}
 
+	@description = "LPS-160159. Assert that it's possible to select a new custom view and selected the default view"
+	@priority = "4"
+	test CanBeSelected {
+		task ("Given FDS sample portlet deployed and given customized tab active") {
+			AssertElementPresent(
+				key_tabName = "Customized",
+				locator1 = "StagingPublishToLive#ACTIVE_TAB_NAME");
+		}
+
+		task ("When open menu to select created custom view") {
+			FDSFilters.editFilters(
+				key_disableFilters = "Blue,Green,Yellow",
+				key_enableFilters = "Red",
+				key_filter = "Color");
+
+			FDSCustomView.createNewView(key_nameView = "Custom Test");
+
+			Click(locator1 = "FrontendDataSet#SELECT_CUSTOM_VIEW");
+
+			Click(
+				key_itemName = "Save View",
+				locator1 = "FrontendDataSet#SELECT_OPTION_HEADER");
+		}
+
+		task ("Then option to select default view is available") {
+			Click(locator1 = "FrontendDataSet#SELECT_CUSTOM_VIEW");
+
+			Click(
+				key_itemName = "Default View",
+				locator1 = "FrontendDataSet#SELECT_OPTION_CUSTOM_VIEW");
+
+			AssertElementPresent(
+				key_wikiNodeName = "Default View",
+				locator1 = "Wiki#NODE_NAME");
```

Comment:
Using a path named Wiki looks unrelated to FDS. Need to create separate path that is either has a general name or specific to FDS.